libs/models/race.ts

```diff
@@ -0,0 +1,24 @@
+import { PlayerWithLogs } from './player';
+
+export interface Race {
+  /** Unique race id */
+  id: string;
+
+  /** Race text which players are typing  */
+  text: string;
+
+  /** Race timeout timer which calculated by the server */
+  timeout: number;
+
+  /** Race started timestamp from the server after player pressed Play button
+   *
+   * This will be used to caluculate force timeout of the race.
+   */
+  startedTimestamp: number;
+
+  /** Players data for specific race
+   *
+   * can be null before players send race starting message to the server.
```

Comment:
We can initialize as empty arrays right?
Removing the `null` state will help us avoid extra null check conditions in the code base

Reply:
Changed